templates/torrc-sample

```diff
@@ -61,4 +61,12 @@ HiddenServicePort 80 <app-specter-desktop-ip>:<app-specter-desktop-port>
 HiddenServiceDir /var/lib/tor/app-btcpay-server
 HiddenServicePort 80 <app-btcpay-server-ip>:<app-btcpay-server-port>
 
+# samourai-dojo Hidden Service
+HiddenServiceDir /var/lib/tor/app-samourai-dojo
+HiddenServicePort 80 <app-samourai-dojo-ip>:<app-samourai-dojo-port>
```

Comment:
Your pointing the Tor hidden service to port `<app-samourai-dojo-port>` (3005) when actually it's running on port 80 on the container and only mapped to port 3005 on the local network.

https://github.com/getumbrel/umbrel/pull/461/files#diff-fed5106248a13f4a034f9b50916d65709d37a2a8c62511565425891fefc38574R120

Reply:
Good spot, that's why the hidden service didn't work.

Reply:
Fixed in afdffbe.